Error when building geqr2 example via make

The geqr2 example's README says it can be built with cmake or make. I populated make.inc with my installation information, but when I tried to run make I got the following output
```
(base) riley@borneophant:~/Documents/Research/protorandlapack/cpp/extern/tlapack/examples/geqr2$ make
g++ -I"/opt/tlapack/include" -Wall -pedantic -c -o example_geqr2.o example_geqr2.cpp
g++ -I"/opt/tlapack/include" -Wall -pedantic -o example_geqr2 example_geqr2.o -L"/opt/tlapack/lib" -ltblas
/usr/bin/ld: cannot find -ltblas
collect2: error: ld returned 1 exit status
```

I was able to get cmake to work. Is building this example with make still supported? If so, what's "tblas"?

--- a/.gitignore
+++ b/.gitignore
@@ -29,3 +29,9 @@ build*
 
 # Visual Studio Code files
 .vscode
+
+# Temporary files
+*.swp
+
+# Other
+notes/*

--- a/examples/cwrapper_gemm/make.inc
+++ b/examples/cwrapper_gemm/make.inc
@@ -4,6 +4,6 @@ tlapack_inc = /usr/local/include
 tlapack_lib = /usr/local/lib
 
 CXXFLAGS = -I$(tlapack_inc) -Wall -pedantic
-LDFLAGS  = -L$(tlapack_lib) -ltblas_c -ltblas
+LDFLAGS  = -L$(tlapack_lib) -ltblas_c
 
 LD = g++

--- a/examples/fortranModule_caxpy/make.inc
+++ b/examples/fortranModule_caxpy/make.inc
@@ -1,9 +1,9 @@
 #-------------------------------------------------------------------------------
 # <T>LAPACK library
-tlapack_inc     = /usr/local/include
-tlapack_lib     = /usr/local/lib
+tlapack_inc = /usr/local/include
+tlapack_lib = /usr/local/lib
 
 FFLAGS = -I$(tlapack_inc) -Wall -pedantic
-LDFLAGS  = -L$(tlapack_lib) -ltblas_fortran -ltblas_c -ltblas -lgfortran
+LDFLAGS  = -L$(tlapack_lib) -ltblas_fortran -ltblas_c -lgfortran
 
 LD = g++

--- a/examples/fortranWrapper_ssymm/Makefile
+++ b/examples/fortranWrapper_ssymm/Makefile
@@ -13,7 +13,7 @@ example_fortranWrapper_ssymm: example_fortranWrapper_ssymm.o
 
 .PHONY: all
 
-%.o: %.f90 constants.mod
+%.o: %.f90 $(tlapack_inc)/constants.mod
 	$(FC) $(FFLAGS) -c -o $@ $<
 
 clean:

--- a/examples/fortranWrapper_ssymm/make.inc
+++ b/examples/fortranWrapper_ssymm/make.inc
@@ -4,6 +4,6 @@ tlapack_inc     = /usr/local/include
 tlapack_lib     = /usr/local/lib
 
 FFLAGS = -I$(tlapack_inc) -Wall -pedantic
-LDFLAGS  = -L$(tlapack_lib) -ltblas_fortran -ltblas_c -ltblas -lgfortran
+LDFLAGS  = -L$(tlapack_lib) -ltblas_fortran -ltblas_c -lgfortran
 
 LD = g++

--- a/examples/gemm/make.inc
+++ b/examples/gemm/make.inc
@@ -4,4 +4,4 @@ tlapack_inc = /usr/local/include
 tlapack_lib = /usr/local/lib
 
 CXXFLAGS = -I$(tlapack_inc) -Wall -pedantic # -DUSE_MPFR
-LDFLAGS  = -L$(tlapack_lib) -ltblas # -lmpfr -lgmp
+LDFLAGS  = -L$(tlapack_lib) # -lmpfr -lgmp

--- a/examples/geqr2/make.inc
+++ b/examples/geqr2/make.inc
@@ -1,7 +1,7 @@
 #-------------------------------------------------------------------------------
 # <T>LAPACK library
-tlapack_inc = /usr/local/include
-tlapack_lib = /usr/local/lib
+tlapack_inc = /opt/tlapack/include #/usr/local/include
+tlapack_lib = /opt/tlapack/lib #/usr/local/lib
 
 CXXFLAGS = -I$(tlapack_inc) -Wall -pedantic
-LDFLAGS  = -L$(tlapack_lib) -ltblas
+LDFLAGS  = -L$(tlapack_lib)

--- a/examples/mdspan/make.inc
+++ b/examples/mdspan/make.inc
@@ -4,4 +4,4 @@ tlapack_inc = /usr/local/include
 tlapack_lib = /usr/local/lib
 
 CXXFLAGS = -I$(tlapack_inc) -Wall -pedantic
-LDFLAGS  = -L$(tlapack_lib) -ltlapack
+LDFLAGS  = -L$(tlapack_lib)

